src/test/java/domain/CarsTest.java

```diff
@@ -4,74 +4,41 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
-import org.junit.jupiter.params.provider.ValueSource;
-import org.springframework.test.util.ReflectionTestUtils;
 
 import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 class CarsTest {
-    @DisplayName("자동차 대수를 입력했을 때 Cars가 정상적으로 생성되는지 테스트")
-    @ParameterizedTest
-    @ValueSource(strings = {"a,b,c", "a  ,b#, c", "abcde,12345,!@#$%", "-12#4, a1cd,!@s34", "a, b", "ad", "a,b,"})
-    void constructCarsTest(String carsName) {
-        Cars cars = new Cars(carsName);
-        List<Car> carList = (List<Car>) ReflectionTestUtils.getField(cars, "cars");
-        assertThat(carList).hasSize(carsName.split(",").length);
-    }
 
-    @DisplayName("자동차 대수를 입력했을 때 Cars가 정상적으로 생성되는지 테스트 - 실패 케이스")
+    @DisplayName("자동차 리스트 이동 테스트")
```

Comment:
이 설명만으로 이 테스트가 무엇을 의도하는지 알 수 있을까요?

```suggestion
    @DisplayName("모든 자동차는 이동이 가능한 경우 전진 후 변경된 위치 값을 가진다.")
```

Reply:
네이밍이 중요하네요. 처음 본다고 생각했을 때 무엇을 테스트하는지 한 눈에 알지 못할것같아요